.github/workflows/pull_request.yml

```diff
@@ -116,7 +119,12 @@ jobs:
         with:
           version: v0.4.2
 
-      - name: Build Image
-        env:
-          IMG_RESULT: cache
-        run: make docker.buildx.csi-driver
+      - name: Build & Push Image
```

Comment:
push for pull request?

Reply:
the `push:` is set to false. I will rename the step to avoid confusion.